### PR TITLE
(DOCSP-38230): Verify Compass signatures

### PR DESCRIPTION
**Pull request by jeff-allen-mongo**
## DESCRIPTION

Add docs to verify Compass package signatures.

## STAGING

[Verify Integrity of Compass Packages](https://preview-mongodbjeffallenmongo.gatsbyjs.io/compass/DOCSP-38230/install/verify-signatures/) (and subpages)

## JIRA

https://jira.mongodb.org/browse/DOCSP-38230

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=667b0aeaebce19f31403513b

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: snooty.toml [hunk 1]
```diff
@@ -30,7 +30,7 @@ toc_landing_pages = [
 
 [constants]
 download-page = "`downloads page <https://www.mongodb.com/try/download/compass>`__"
-current-version = "1.43.0"
+current-version = "1.43.2"
```
Comment (mabaasit) [inline snooty.toml:33]: Released today
```suggestion
current-version = "1.43.3"
```

File: source/install/verify-signatures/gpg.txt [hunk 2]
```diff
@@ -0,0 +1,90 @@
+.. _compass-verify-signatures-gpg:
+
+==========================================
+Verify Packages with GPG (Linux and macOS)
```
Comment (mabaasit) [inline source/install/verify-signatures/gpg.txt:4]: In Compass, we can also verify archive (windows) using gpg! Even nupkg. So maybe we should 
```suggestion
Verify Packages with GPG
```

File: source/includes/verification-gpg-results.rst [hunk 3]
```diff
@@ -0,0 +1,17 @@
+If the key imports successfully, the command returns:
+
+.. code-block:: sh
+    :copyable: false
+
+    gpg: key CEED0419D361CB16: public key "MongoDB Compass Signing Key <compass@mongodb.com>" imported
+    gpg: Total number processed: 1
+    gpg:              imported: 1
```
Comment (mabaasit) [inline source/includes/verification-gpg-results.rst:8]: missing space for aligning `1`?  or else this is how it looks in web
image

File: source/install/verify-signatures/gpg.txt (hunk 2 above)
Comment (jeff-allen-mongo) [reply source/install/verify-signatures/gpg.txt:4]: Cool! That's good to know.

File: source/includes/verification-gpg-results.rst (hunk 3 above)
Comment (jeff-allen-mongo) [reply source/includes/verification-gpg-results.rst:8]: Ach I've been struggling getting these aligned 😅 fixed!

**Comment by jeff-allen-mongo** [comment]
@mabaasit thanks for the review! I made the suggested edits and this is ready for another look.

**Review by mabaasit — APPROVED** [review]

File: source/install/verify-signatures/disk-images.txt [hunk 4]
```diff
@@ -0,0 +1,43 @@
+.. _compass-verify-signatures-disk-image:
+
+============================================
+Verify Packages with Disk Image Verification
+============================================
+
```
Comment (sarah-olson-mongodb) [inline source/install/verify-signatures/disk-images.txt:6]: (spacing)
There's an extra line between the title and the first para, in the [HTML](https://preview-mongodbjeffallenmongo.gatsbyjs.io/compass/DOCSP-38230/install/verify-signatures/disk-images/).

File: source/install/verify-signatures/gpg.txt [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+----------------
+
+.. include:: /includes/verify-signatures-before-you-begin.rst
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+
```
Comment (sarah-olson-mongodb) [inline source/install/verify-signatures/gpg.txt:31]: Lol --- Good call Vale.

File: source/install/verify-signatures/gpg.txt [hunk 6]
```diff
@@ -0,0 +1,90 @@
+.. _compass-verify-signatures-gpg:
+
+========================
+Verify Packages with GPG
+========================
+
```
Comment (sarah-olson-mongodb) [inline source/install/verify-signatures/gpg.txt:6]: (thanks!!)
Thanks a ton for incorporating my edits into this PR as well!

**Review by sarah-olson-mongodb — APPROVED** [review]
Thanks @jeff-allen-mongo! LGTM after fixing one weird thing in the HTML.

File: source/install/verify-signatures/disk-images.txt (hunk 4 above)
Comment (jeff-allen-mongo) [reply source/install/verify-signatures/disk-images.txt:6]: Hmm I don't think I'm seeing this in my staged view? Could you maybe show a screenshot?

Comment (jeff-allen-mongo) [reply source/install/verify-signatures/disk-images.txt:6]: @sarah-olson-mongodb I can't see anything off with the rst here, so I'm going to chalk this up to an odd staging behavior.

**Pull request merged**